app/src/main/java/com/example/tasky/event/presentation/Event.kt

```diff
@@ -0,0 +1,152 @@
+package com.example.tasky.event.presentation
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.safeDrawingPadding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.navigation.NavController
+import com.example.tasky.R
+import com.example.tasky.common.domain.util.convertMillisToDate
+import com.example.tasky.common.presentation.CheckboxHeader
+import com.example.tasky.common.presentation.DateLineItem
+import com.example.tasky.common.presentation.GrayDivider
+import com.example.tasky.common.presentation.RightCarrotIcon
+import com.example.tasky.event.presentation.components.EmptyPhotos
+
+@Composable
+fun EventRoot(
+    navController: NavController,
+    date: Long,
+    eventViewModel: EventViewModel = hiltViewModel()
+) {
+//    val viewState by eventViewModel.viewState.collectAsStateWithLifecycle()
+    EventContent(
+        dateOnToolbar = date.convertMillisToDate(),
+        onToolbarAction = {}
+    )
+}
```

Comment:
One of the tougher decisions to make for this project is whether to use one screen per agenda item type or one screen for all agenda items. 2 years ago, I was also team single screen per type, but my mind has changed :D

My concern was that having a single screen makes it harder to read and update, but it actually got a lot easier:
If you look at the UI hierarchy of the items, reminder and task screens are almost identical while the event screen also has the same elements with a few extra composables. 

The complete root screen structure is no more than this:
```
HeaderSection()
TitleSection()
DescriptionSection()
if(item is Event) {
    PhotoSection()
}
StartDateSection()
if(item is Event) {
    EndDateSection()
}
ReminderSection()
if(item is Event) {
    AttendeeSection()
}
DeleteButton()
```
As long as you extract the UI into single components, it remains very readable. Same goes for the underlying ViewModel, you will save so much code by just having one ViewModel, since the logic of what all items have in common only needs to be written once. And if you'd extend that screen with another type of agenda item, that would also be easier and wouldn't require you to duplicate most of this to another screen.